### ISSUE TITLE
[A11y] Enable A11y Tests: NavigationDrawerActivityTest 

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#3248]
Multiple tests are failing on #3246 with following error:

<img width="1551" alt="Screenshot 2021-05-25 at 7 02 52 PM" src="https://user-images.githubusercontent.com/9396084/119506877-f432b600-bd8b-11eb-8247-806352fd42d4.png">

**Comment by anandwana001** [comment]
Are all these espresso or robolectric?
Also, it looks like they are failing due to accessibility. 
Are they failing on develop too?

**Comment by rt4914** [comment]
@anandwana001 All of these are espresso and also only when we enable `AccessibilityChecks` otherwise all of these are passing on current develop.
